x-pack/solutions/security/packages/kbn-securitysolution-utils/date_math.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './src/date_math';
```

Comment:
Why do we export date math and time duration utilities via their own files, but not via the `index.ts`?